package/MDAnalysis/coordinates/base.py

```diff
@@ -2045,8 +2045,28 @@ def add_transformations(self, *transformations):
 
         """
 
+        # Checks if list is passed
+        if isinstance(transformations, list):
+            six.rais_from(
+                TypeError(
+                    "You've passed a list!"
+                    "Please remove the [] and try passing it again"
+                )
+            )
+            pass
+
+        # Performs the transform
         try:
             self.transformations = transformations
+        except TypeError:
+            # Check if nested list, if yes then 'remove' one layer of [] and apply transform
+            # Else, raises error asking for correct user input type
+            six.raise_from(
+                TypeError(
+                    "It seems as though you've passed an incompable object type!"
+                    "Please check the type being passed again"
+                )
+                   
```

Comment:
There is a missing `)` here, this is why tests fail.

Reply:
I've added the ')' now. I'm extremely sorry for not seeing it earlier. I'll read up on the article to run tests locally before pushing code next time.
Thanks alot for your kind help and support!